src/main/java/core/org/akaza/openclinica/dao/EventCRFSDVFilter.java

```diff
@@ -49,18 +56,22 @@ public String execute(String criteria) {
     }
 
     private String buildCriteria(String criteria, String property, Object value) {
-        value = StringEscapeUtils.escapeSql(value.toString());
-        if (value != null) {
+        if(StringUtils.isNotBlank(value.toString())){
```

Comment:
if value is null, this would throw an NPE. You should first check if value is null before checking for blank string.

Reply:
Made changes as per the comment